src/style.css

```diff
@@ -1,3 +1,115 @@
+/* Configuração geral da fonte */
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+/* Estilização do cabeçalho */
+header {
+  display: flex;
+  align-items: center;
+  width: 1024px;
+  padding: 0 20px;
+  background-color: white;
+  height: 60px;
+  position: relative;
+  margin: 0 auto;
+  justify-content: space-between;
+}
+
+/* Estilização da imagem do logo */
+.logo img {
+  width: 40px;
+  height: 40px;
+  position: relative; /* Para que as propriedades top e left tenham efeito */
+  top: 10px; /* Ajuste a posição do logo conforme necessário */
+  left: 50px; /* Ajuste a posição do logo conforme necessário */
```

Comment:
Avoid using tag names for styling. Instead, use class selectors to apply styles. For example, use `.logo-img` instead of `.logo img`.